src/ln/channelmonitor.rs

```diff
@@ -1165,6 +1209,7 @@ impl ChannelMonitor {
 									value: htlc.amount_msat / 1000, //TODO: - fee
 								}),
 							};
+							single_htlc_tx.output[0].value -= fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::HighPriority) * Self::get_claim_tx_weight(vec![if htlc.offered { InputDescriptors::RevokedOfferedHTLC } else { InputDescriptors::RevokedReceivedHTLC }], &single_htlc_tx.output[0].script_pubkey) / 1000;
```

Comment:
Maybe make get_claim_tx_weight take a &[] instead of a Vec-by-move? That way you wouldn't have to create the heap allocation here and LLVM could probably optimize it almost completely away.

Reply:
Good idea, done